src/boostvqe/models/dbi/double_bracket_evolution_oracles.py

```diff
@@ -307,3 +307,43 @@ def circuit(self, t_duration, steps=None, order=None):
             steps=steps,
             order=order,
         )
+
+
+@dataclass
+class tfim_EvolutionOracle(EvolutionOracle):
+    steps: int = None
+    B_a: float = None
+
+    def circuit(self, a, t_duration, B_a, steps=None, order=None):
```

Comment:
```suggestion
    def circuit(self, t_duration, steps=None, order=None):
```
This class must implement `eo.circuit(0.1)` without requiring other parameters.

What is `a`?

Reply:
I use the notation in the paper where a is an index and  B_a is another parameter.

$$H^{(a)}=X_{a}X_{a+1}+B_{a}Z_{a}$$

For our purpose, do we intend to sum up all the a?